pkg/api/v1alpha1/tinkerbelldatacenterconfig_types.go

```diff
@@ -9,7 +9,10 @@ import (
 // TinkerbellDatacenterConfigSpec defines the desired state of TinkerbellDatacenterConfig
 type TinkerbellDatacenterConfigSpec struct {
 	// Important: Run "make generate" to regenerate code after modifying this file
-	TinkerbellIP string `json:"tinkerbellIP"`
+	TinkerbellIP           string `json:"tinkerbellIP"`
+	TinkerbellCertURL      string `json:"tinkerbellCertURL"`
+	TinkerbellGRPCAuth     string `json:"tinkerbellGRPCAuth"`
+	TinkerbellPBnJGRPCAuth string `json:"tinkerbellPBnJGRPC"`
```

Comment:
nit: just for consistency
```suggestion
	TinkerbellPBnJGRPCAuth string `json:"tinkerbellPBnJGRPCAuth"`
```